src/net.cpp

```diff
@@ -1454,7 +1454,11 @@ void ThreadDNSAddressSeed()
                     found++;
                 }
             }
-            addrman.Add(vAdd, CNetAddr(seed.name, true));
+
+            // TODO: It doesn't make sense to define an arbitrary IP as the source of others.
+            // Instead, a dummy range should be created to identify seed resolves.
+            if (!vIPs.empty())
+                addrman.Add(vAdd, vIPs[0]);
```

Comment:
@sipa To clarify, I meant that each dns seed should have a corresponding stable "source" dummy ip. Something like this: https://github.com/theuni/bitcoin/commit/792b0f5da618ea51ecd7b21db633faa6743c1e68

I see now that the second resolve is happening on seed.name rather than seed.host, so the resolve isn't actually arbitrary. That was just poor reading on my account.

I'll add back the second resolve, and we can revisit the dummy later.